cms/static/js/views/settings/main.js

```diff
@@ -2,469 +2,503 @@ define(['js/views/validation', 'codemirror', 'underscore', 'jquery', 'jquery.ui'
     'js/models/uploads', 'js/views/uploads', 'js/views/license', 'js/models/license',
     'common/js/components/views/feedback_notification', 'jquery.timepicker', 'date', 'gettext',
     'js/views/learning_info', 'js/views/instructor_info', 'edx-ui-toolkit/js/utils/string-utils'],
-       function(ValidatingView, CodeMirror, _, $, ui, DateUtils, FileUploadModel,
```

Comment:
This files formatting was so over the place that it was hard to work in. I think this and other JS files were a refactor gone wrong (note the comments being all skewed). I've formatted the original file, diffed them, and made a gist so you can see the actual content changes: https://gist.github.com/tuchfarber/9f7000078f12c295b5e8a6b4a9516afe